libcxx/test/std/input.output/string.streams/concepts.h

```diff
@@ -0,0 +1,17 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef TEST_STD_INPUTOUTPUT_STRINGSTREAMS_CONCEPTS_H
+#define TEST_STD_INPUTOUTPUT_STRINGSTREAMS_CONCEPTS_H
+
+template <typename S, typename T>
+concept HasStr = requires(S s, const T& sv) {
```

Comment:
please use a more descriptive name, for example `is_valid_argument_for_str_member`. That makes it clearer at the call side what this does. Especially since this concept resides in a different file.